io/plugins/eu.esdihumboldt.hale.io.jdbc.mssql/src/eu/esdihumboldt/hale/io/jdbc/mssql/MsSqlGeometries.java

```diff
@@ -221,19 +221,20 @@ public void run() {
 					}
 				});
 				crsDef = result.get();
-			}
-			// SRS Code
-			String srsName = CRS.toSRS(crsDef.getCRS());
-			if (srsName != null) {
-				final int index = srsName.lastIndexOf(':');
-				String authorityName = null;
-				String authorizedId = null;
-				if (index > 0) {
-					authorityName = srsName.substring(0, index);
-					authorizedId = srsName.substring(index + 1).trim();
+
+				// saving in cache
+				String srsName = CRS.toSRS(crsDef.getCRS());
+				if (srsName != null) {
+					final int index = srsName.lastIndexOf(':');
+					String authorityName = null;
+					String authorizedId = null;
+					if (index > 0) {
+						authorityName = srsName.substring(0, index);
+						authorizedId = srsName.substring(index + 1).trim();
+					}
+					// we don't need wkt.
+					SRSUtil.addSRSinCache(srId, authorityName, authorizedId, null);
 				}
-				String wkt = crsDef.getCRS().toWKT();
-				SRSUtil.addSRSinCache(srId, authorityName, authorizedId, wkt);
```

Comment:
Yes. But some how the same error was not logged in to my error log (Otherwise i could have solved it) So I didn't bother about debugging it then. but today, when I debug it then I came to know this error.